azure-pipelines.yml

```diff
@@ -23,6 +23,12 @@ steps:
     echo '##vso[task.prependpath]$(GOROOT)/bin'
   displayName: 'Set up the Go workspace'
 
+- script: |
+    go get github.com/gobuffalo/packr/packr
```

Comment:
I am ok with either here or in a follow-on PR making another target like `make get-deps` that does all of this.

Reply:
Added `get-deps`.  Each target that requires it now calls it, so no real need for a separate bootstrap script here; therefore, I've removed.  Let me know if I should add back in to make explicit...